superset/views/geocoding.py

```diff
@@ -161,12 +162,12 @@ def geocode(self) -> Response:
             return json_error_response(e.args[0], status=500)
 
         try:
-            data = self._geocode(data)
-        except Exception as e:
-            self.logger.exception(f"Exception when geocoding data {e}")
-            if not save_on_stop_geocoding:
-                self.stats_logger.incr("geocoding_failed")
-                return json_error_response(e.args[0])
+            data = self._geocode(data, "maptiler")
+        except NoAPIKeySuppliedException:
+            message = f"No Key was supplied for specified Geocoding API"
```

Comment:
If possible add which geocoder

Reply:
Noted and will do so in the refactoring branch